utils/pytorch.py

```diff
@@ -123,19 +121,95 @@ def print_performance_metrics(self, batch_size):
             self.__start_times, self.__finish_times, self.__times_invoked, batch_size
         )
 
-        dump_dir = os.environ.get("RESULTS_DIR")
-        if dump_dir is not None and len(self.__start_times) > 2:
-            with open(f"{dump_dir}/meta_{os.getpid()}.json", "w") as f:
-                json.dump({"batch_size": batch_size}, f)
-            with open(f"{dump_dir}/{os.getpid()}.csv", "w") as f:
-                writer = csv.writer(f)
-                writer.writerow(self.__start_times[2:])
-                writer.writerow(self.__finish_times[2:])
+        bench_utils.dump_csv_results(batch_size, self.__start_times, self.__finish_times)
 
         if self.__is_profiling:
             print(self.__profile.key_averages().table(sort_by='cpu_time_total', row_limit=50))
             torch._C._aio_profiler_print()
         return perf
 
+
+class PyTorchRunnerV2:
```

Comment:
It's not that much different from V1. What's the plan here? Do we want to eventually move all of the models to use V2 and it's just a temporary measure to have both of them at the same time?